src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/NavMenu.razor.css

```diff
@@ -58,5 +58,9 @@
     .collapse {
         /* Never collapse the sidebar for wide screens */
         display: block;
+        
+        /* Allow sidebar to scroll for tall menus */
+        height: calc(100vh - 3.5rem);
+        overflow-y: auto;
```

Comment:
Hmm, I wonder if it may seem a bit odd to have this logic within the `collapse` class. I see why you did it given this is the only applicable class for the column:

https://github.com/dotnet/aspnetcore/blob/e74e5d068099868f4a652f913be0a5c4d083b33b/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Shared/NavMenu.NoGraphOrApi.razor#L33

Will wait for others to chime in on this.

Reply:
What about putting it onto the `.flex-column`?